.jscsrc

```diff
@@ -38,7 +38,7 @@
    "disallowYodaConditions": true,
    "maximumLineLength": 120,
    "requireBlocksOnNewline": true,
-   "requireCamelCaseOrUpperCaseIdentifiers": true,
+   "requireCamelCaseOrUpperCaseIdentifiers": false,
```

Comment:
I see, good point. I think that in regard of `_request` we can rename it wherever it's used because it isn't even private anymore. In regard of page, we can add an _ad hoc_ exception on a specific line as we do with Istanbul. What do you think?